.travis/install.sh

```diff
@@ -19,7 +19,5 @@ if [[ "$(uname -s)" == 'Darwin' ]]; then
     pyenv activate conan
 fi
 
-pip install conan --upgrade
-pip install conan_package_tools
-
+pip install conan_package_tools bincrafters_package_tools
```

Comment:
this won't upgrade conan installation. is this intended?

Reply:
good catch!!! fixing it!!!